plugins/module_utils/dnac.py

```diff
@@ -17,6 +17,7 @@
 from abc import ABCMeta, abstractmethod
 try:
     import logging
+    import ipaddress
```

Comment:
Do we need to import "re" module?
import re

Reply:
Yes, because we are using it for password validation and other api's as well.